.konflux/operator/konflux.Dockerfile

```diff
@@ -0,0 +1,45 @@
+ARG OPERATOR_VERSION
+
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:v1.23@sha256:8c881edc745eeb6ea246f1bac55b09b5ae5dc16f7ed9d0d73b5ffda28e582bab as builder
```

Comment:
why do you pin specific image (`@sha256:8c881edc745eeb6ea246f1bac55b09b5ae5dc16f7ed9d0d73b5ffda28e582bab`)

Reply:
Konflux uses Renovate to check for and perform dependency updates. for mintmaker-renovate  updates the container pull spec must be a pinned digest or a tag.
see example https://github.com/openshift-kni/lifecycle-agent/blob/main/.konflux/konflux_build_args.conf